.github/workflows/publish-npm-packages.yml

```diff
@@ -48,11 +48,56 @@ jobs:
         run: |
           EXTENSIONS=$(node ${{ env.SCRIPTS_DIR }}/get-extensions.js)
           echo "::set-output name=extensions::{\"include\":${EXTENSIONS}}"
-  publish:
+  publish-core:
     if: github.repository == 'ampproject/amphtml'
     environment: bento_publishing
     needs: setup
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.events.inputs.amp-version }}
+      - uses: actions/setup-node@v2
+        with:
+          registry-url: https://registry.npmjs.org
+          node-version: 16
+      - name: Get latest scripts
+        run: |
+          wget -q "${{ env.SCRIPTS_REPO }}/${{ env.SCRIPTS_DIR }}/utils.js" -O ${{ env.SCRIPTS_DIR }}/utils.js
+          wget -q "${{ env.SCRIPTS_REPO }}/${{ env.SCRIPTS_DIR }}/get-core-version.js" -O ${{ env.SCRIPTS_DIR }}/get-core-version.js
+          wget -q "${{ env.SCRIPTS_REPO }}/${{ env.SCRIPTS_DIR }}/build-npm-binaries.js" -O ${{ env.SCRIPTS_DIR }}/build-npm-binaries.js
+      - name: Build package
+        run: |
+          npm ci
+          node ${{ env.SCRIPTS_DIR }}/build-npm-binaries.js
+          cd ./src/bento/core
```

Comment:
@esther each step is re-initialized at the root of the repo, right? That's what it sounded like from the circleci, but just wanted to double check

Reply:
Each job is initialized, and steps are run on the same machine. Btw this isn't circleci but github actions. This page is super helpful https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions

Reply:
ohh my mistake. still bit totally clear if github action steps are re-initialized to the root of the repo for each step or I'd need to `cd ../../../` back to root. I'm assuming the former, but I'll change dir back to root just to be safe

Reply:
You can also test on your local fork

Reply:
Confirmed on my fork that different steps of the same job do get reinitialized back to the root of the repo: https://github.com/kvchari/amphtml/runs/5383481179?check_suite_focus=true#step:6:12 (i logged the pwd, changed dir, logged pwd again, then did the same thing in the next step. noticed that the different steps start at the same repo root)

Reply:
removing the unnecessary cd ../../.. and resolving